lib/tasks/migrate_to_pg.rake

```diff
@@ -1,48 +1,63 @@
 namespace :migrate_to_pg do
   desc 'Migrate Config from Mongo to Postgres'
   task config: :environment do
-    pg = Config
-    mongo = MongoConfig
-    migrate_model(pg, mongo)
+    ActiveRecord::Base.transaction do
+      pg = Config
+      mongo = MongoConfig
+      migrate_model(pg, mongo)
+    end
   end
 
   task event: :environment do
-    pg = Event
-    mongo = MongoEvent
-
-    extra_transform = Proc.new do |attrs|
-      event_type_mapping = {
-        "Reached patient" => :reached_patient,
-        "Couldn't reach patient" => :couldnt_reach_patient,
-        "Left voicemail" => :left_voicemail,
-        "Pledged" => :pledged,
-        "Unknown action" => :unknown_action
-      }
-      attrs['event_type'] = event_type_mapping[attrs['event_type']]
-      attrs
+    ActiveRecord::Base.transaction do
+      pg = Event
+      mongo = MongoEvent
+
+      extra_transform = Proc.new do |attrs|
+        event_type_mapping = {
+          "Reached patient" => :reached_patient,
+          "Couldn't reach patient" => :couldnt_reach_patient,
+          "Left voicemail" => :left_voicemail,
+          "Pledged" => :pledged,
+          "Unknown action" => :unknown_action
+        }
+        attrs['event_type'] = event_type_mapping[attrs['event_type']]
+        attrs
+      end
+
+      migrate_model(pg, mongo, extra_transform)
     end
+  end
 
-    migrate_model(pg, mongo, extra_transform)
+  task clinic_user_patient: :environment do
+    ActiveRecord::Base.transaction do
+      # Start by moving clinics
+      pg = Clinic
+      mongo = MongoClinic
+      extra_transform = Proc.new do |attrs, obj|
+        attrs['mongo_id'] = obj['_id'].to_s
```

Comment:
So that we can preserve existing relationships we're adding a mongo_id field to stuff that relates to other stuff. This is part of the reason why we're doing Clinic first - it doesn't depend on much stuff compared to Patient (though we could also do User at this juncture).